tests/components/cloud/test_http_api.py

```diff
@@ -1943,7 +1943,8 @@ async def mock_empty_info(hass: HomeAssistant) -> dict[str, Any]:
     )
 
     now = dt_util.utcnow()
-    freezer.move_to(datetime.datetime.fromisoformat("2025-02-10T12:00:00.0+00:00"))
+    tz = now.astimezone().tzinfo
```

Comment:
Let's explain this, maybe:
```suggestion
    # The logging is done with local time according to the system timezone, set the
    # fake time to noon local time
    tz = now.astimezone().tzinfo
```